spec/controllers/send_application_controller_spec.rb

```diff
@@ -34,6 +34,15 @@
           current_app.reload.email
         }.from("test@example.com").to("new_email@example.com")
       end
+
+      it "creates a SendApplicationJob" do
```

Comment:
this is kind of a lame test because it's all stubs. Thoughts on a better way to test that this is called?